examples/gno.land/p/gnoland/commondao/proposal.gno

```diff
@@ -0,0 +1,174 @@
+package commondao
+
+import (
+	"errors"
+	"std"
+	"time"
+)
+
+// DefaultQuorum defines the default quorum required to tally proposal votes.
+const DefaultQuorum = 0.34 // 34%
```

Comment:
shouldn't be 2/3?

Reply:
It could be. The though behind a less restrictive value was to make it more flexible by default. Users can always define a higher quorum per proposal type if they want to override it.

To be honest I chose a less restrictive value but using 2/3 by default might be the way to go.

Reply:
Update, this const has been removed in c4ec22a, so quorum is optional and must be checked by devs